src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,15 @@
+import java.util.Map;
+
+public class ReturnOperation implements OperationHandler {
+    private final Map<String, Integer> fruitStorage;
```

Comment:
The `fruitStorage` should not be directly accessed in the strategy. Instead, consider using a service that will handle the interaction with the storage.